arches/app/models/fields/i18n.py

```diff
@@ -22,6 +22,10 @@ def _parse(self, value, lang, use_nulls):
         ret = {}
 
         if isinstance(value, str) and value != "null":
+            # fix for issue #9623 - using double quotation marks in i18n input
```

Comment:
You can remove this line. We can always trace back to the ticket through a git blame.